testsuite/MDAnalysisTests/core/test_groups.py

```diff
@@ -1483,9 +1489,13 @@ def test_get_non_topattr(self, universe):
     def test_unwrap_without_bonds(self, universe):
         with pytest.raises(NoDataError) as exc:
             universe.atoms.unwrap()
-        err = ('AtomGroup.unwrap() not available; '
-               'this requires Bonds')
-        assert str(exc.value) == err
+        expected_message = (
+            "AtomGroup.unwrap() not available; this AtomGroup lacks defined bonds. "
+            "To resolve this, you can either:\n"
+            "1. Guess the bonds at universe creation using `guess_bonds = True`, or\n"
+            "2. Create a universe using a topology format where bonds are pre-defined."
+        )
+        assert str(exc.value) == expected_message
```

Comment:
Given that pytest.raises has the ability to also check the message, we would like to use its functionality as it keeps the tests more compact. Could you please rewrite similar to how you wrote your own code with `match=...`?